src/pages/Chat/ChatRoom/ChatRoom.tsx

```diff
@@ -83,69 +95,75 @@ function ChatRoom({
   const disableInput = (): boolean => {
     if (
       userRole === "client" &&
```

Comment:
can user roles be enum? I believe it should be already somwhere in project, or create a new one. Don't compare to string